src/Test/StateMachine/Parallel.hs

```diff
@@ -662,16 +663,20 @@ exists' xs p = exists xs p
 prettyParallelCommands :: (MonadIO m, Rank2.Foldable cmd)
                        => (Show (cmd Concrete), Show (resp Concrete))
                        => ParallelCommands cmd resp
+                       -> Maybe GraphOptions
                        -> [(History cmd resp, Logic)] -- ^ Output of 'runParallelCommands'.
                        -> PropertyM m ()
-prettyParallelCommands cmds =
-  mapM_ (\(hist, l) -> printCounterexample hist (logic l) `whenFailM` property (boolean l))
+prettyParallelCommands cmds mGraphOptions hist =
```

Comment:
Could we rename this to `prettyParallelCommandWithOpts` and define a backwards compatible `prettyParalellCommands` in terms of the `WithOpts` one where we pass `Nothing` as options? That way we couldn't break the old API.